lib/src/models/album.dart

```diff
@@ -100,9 +89,9 @@ class AlbumSimple extends Object {
   String releaseDate;

```

Comment:
@rinukkusu should this not be a `Date` object? What do you think?

Reply:
You're right, but I don't know if `Date` can parse all these 3 "precisions" correctly.

> Depending on the precision, it might be shown as "1981" or "1981-12".

Can you please try it and check with the other 2 precisions in your test?

Reply:
Yeah, it doesn't work. For `expect(album.releaseDate, DateTime(2012));` I get `FormatException: Invalid date format
2012`

Reply:
Sad 😄 
LGTM otherwise!